PyInstaller/depend/analysis.py

```diff
@@ -126,7 +126,7 @@ def _reset(self, user_hook_dirs):
         self._additional_files_cache = AdditionalFilesCache()
         # Prepend PyInstaller hook dir to user hook dirs.
```

Comment:
Update this comment to explain: command-line hooks (I'm assuming that's what is `user_hook_dirs`) are highest priority, because this list is processed (I'm also assuming) in priority order: lower index = higher priority. Next comes PyInstaller hooks, based on this code. But I thought that entry points hooks should come next? The code in `building/build_main.py` line 201 adds in entry points hooks -- that should probably be fixed as well.